prep-html-links/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <a href="file:///C:/Users/derek/repos/prep-0221-code-solutions/prep-html-links/contact-us/contact-us.html">Click here to go to the contact page</a>
```

Comment:
You want to use a relative path here instead of an absolute path. (`contact-us/contact-us.html`).